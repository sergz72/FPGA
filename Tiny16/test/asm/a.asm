	j start
	reti
start:
	lli SP, $FF80
	jal SP, init
next:
	wfi
	xor A, W
	sw A, 0(X)
	j next

init:
	li X, $8000
	lli A, 0
	lli W, 1
	loadpc -1(SP)
