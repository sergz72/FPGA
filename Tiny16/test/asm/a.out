4002 // jmp 2
0001 // nop
0001 // nop
6551 // mvil r1, 0x55
7AA1 // mvih r1, 0xAA
6AA2 // mvil r2, 0xAA
7552 // mvih r2, 0x55
9042 // mov (r1), r2
B042 // adi r2, 8
9042 // mov (r1), r2
0000 // halt
